### PR TITLE
add decimals to staking rewards displayed

### LINKED ISSUE #661: Fix 0% rounding issue on main landing page
image

This is rounding to 0 places, but now that AKR is lower weekly rewards are below 1%

We should either round to more places (probably need 2) or do something more dynamic like rounding to more places if the rate is below a certain amount (e.g. round to 1 place between 1 and 0.1, round to 2 places between 0.1 and 0.01, etc.

### PR DESCRIPTION
**Pull request by goatsOnDrugs**
## Description
adds 2 decimal places to the staking rewards and change from weekly to monthly

I dont think so. smol change.

## Related Ticket

Resolves #661 

## Checklist

- [x] Building the site with `npm run build-site` works without errors
- [x] Building the app with `npm run build-app` works without errors
- [x] I formatted JS and TS files with running `npm run format-all`

### CODE REVIEW

File: site/pages/index.tsx [hunk 1]
```diff
@@ -16,7 +16,7 @@ export const getStaticProps: GetStaticProps<Props> = async (ctx) => {
   const translation = await loadTranslation(ctx.locale);
   const blockRate = await fetchBlockRate();
   const weeklyStakingRewards = await getStakingRewards({
```
Comment (Atmosfearful) [inline site/pages/index.tsx:18]:
```suggestion
  const monthlyStakingRewards = await getStakingRewards({
```

**Review by Atmosfearful — CHANGES_REQUESTED** [review]
thanks, just a variable name nitpick:

File: site/pages/index.tsx (hunk 1 above)
Comment (goatsOnDrugs) [reply site/pages/index.tsx]: good catch, updated the translation as well.

**Review by Atmosfearful — APPROVED** [review]
ty ty

**Pull request merged**